Bug report: `formatCurrency()` returns wrong format for `de-DE` as `naviator.language`

**Describe the bug** A clear and concise description of what the bug is.

The `formatCurrency()` test is failing on a computer with `navigator.language` set to `de-DE`.  

```
[display] formatCurrency() => ./utils/display_test.ts:34:6
error: AssertionError: Values are not equal.


    [Diff] Actual / Expected


-   5$
+   $5


  throw new AssertionError(message);
        ^
    at assertEquals (https://deno.land/std@0.202.0/assert/assert_equals.ts:53:9)
    at file:///Users/<user>/saaskit/utils/display_test.ts:35:3
```

**To Reproduce** Steps to reproduce the behavior:

Run `deno task test` with the `navigator.language` set to `de-DE` for example.

**Expected behavior** A clear and concise description of what you expected to
happen.

Having the expected correct `$` format however the `navigator.language`. 

**Additional context** Add any other context about the problem here.

This might be an interesting case since for example `€` is handled the other way. There it would be `5€` and not `€5`

--- a/utils/display.ts
+++ b/utils/display.ts
@@ -55,7 +55,7 @@ export function timeAgo(date: Date) {
 
 /**
  * Returns a formatted string based on the given amount of currency and the
- * machine's preferred language.
+ * `en-US` locale. Change the locale for your use case as required.
  *
  * @see {@linkcode Intl.NumberFormat}
  *
@@ -71,7 +71,7 @@ export function formatCurrency(
   currency: string,
 ): string {
   return new Intl.NumberFormat(
-    navigator.language,
+    "en-US",
     {
       style: "currency",
       currency,

